src/units/attack_type.hpp

```diff
@@ -348,12 +357,15 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	}
 private:
 
+	const std::set<std::string> checking_alignment_{"neutral", "lawful", "chaotic", "liminal"};
```

Comment:
This looks like a copy of `unit_alignments`. If this is really needed that should be clearly documented and explained why `values` of the enum cannot be used.

Reply:
removed